xwiki-platform-core/xwiki-platform-livedata/xwiki-platform-livedata-war/src/main/webapp/resources/uicomponents/widgets/liveData.js

```diff
@@ -42,20 +40,29 @@
   'contextPath': $request.contextPath
 })
 #[[*/
+function loadCSS(url)
+{
+  const link = document.createElement('link')
+  link.type = 'text/css'
+  link.rel = 'stylesheet'
+  link.href = url
+  document.getElementsByTagName('head')[0].appendChild(link)
+}
```

Comment:
Should this really be outside the `function` wrapper below? If yes, I feel like this needs a comment explaining why.